qcodes/tests/drivers/keysight_b1500/b1500_driver_tests/test_b1500.py

```diff
@@ -199,10 +199,5 @@ def test_self_calibration_failed(b1500):
 
 
 def test_error_message(b1500):
-    mock_ask = MagicMock()
-    b1500.ask = mock_ask
-    mock_ask.return_value = '0,"No Error."'
-
     response = b1500.error_message()
     assert '0,"No Error."' == response
-    mock_ask.assert_called_once_with(f'ERRX?')
```

Comment:
I guess we are ok with loosing the counter on ask being called?

Reply:
yep. not only the error_message should just work, but init of the instrument (via fixture) should not accumulate errors in the queue